ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -144,8 +148,19 @@ private static boolean isTransformedTouchPointInView(
       localX = localXY[0];
       localY = localXY[1];
     }
-    if ((localX >= 0 && localX < (child.getRight() - child.getLeft()))
-        && (localY >= 0 && localY < (child.getBottom() - child.getTop()))) {
+    float hitSlopTop = 0;
```

Comment:
@kmagiera `TouchDelegate` is weird because it takes precedence over everything else (and also only applies to one child view unless you do extra work) so I think it's fine for `hitSlop` to work differently. I'll add more details to the comment to explain that. Thanks!